src/bno055_driver.cpp

```diff
@@ -181,39 +181,42 @@ rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
   }
 
   rclcpp::Parameter accelerometer_radius_param;
-  if (get_parameter("calibration/accelerometer_radius", accelerometer_radius_param)) {
-      const int64_t accelerometer_radius = accelerometer_radius_param.get_value<int64_t>();
+  if (get_parameter("calibration.accelerometer_radius", accelerometer_radius_param)) {
+    const int64_t accelerometer_radius = accelerometer_radius_param.get_value<int64_t>();
 
-      RCLCPP_INFO(get_logger(), "Setting accelerometer calibration radius");
+    RCLCPP_INFO(get_logger(), "Setting accelerometer calibration radius");
 
-      port_->write_command_.address = BNO055Register::ACC_RADIUS_LSB;
-      port_->write_command_.length = 2;
-      port_->write_command_.data[0] = accelerometer_radius & 0xFF;
-      port_->write_command_.data[1] = (accelerometer_radius >> 8) & 0xFF;
-      port_->write();
+    port_->write_command_.address = BNO055Register::ACC_RADIUS_LSB;
+    port_->write_command_.length = 2;
+    port_->write_command_.data[0] = accelerometer_radius & 0xFF;
+    port_->write_command_.data[1] = (accelerometer_radius >> 8) & 0xFF;
+    port_->write();
   }
 
   rclcpp::Parameter magnetometer_radius_param;
-  if (get_parameter("calibration/magnetometer_radius", magnetometer_radius_param)) {
-      const int64_t magnetometer_radius = magnetometer_radius_param.get_value<int64_t>();
+  if (get_parameter("calibration.magnetometer_radius", magnetometer_radius_param)) {
+    const int64_t magnetometer_radius = magnetometer_radius_param.get_value<int64_t>();
 
-      RCLCPP_INFO(get_logger(), "Setting magnetometer calibration radius");
+    RCLCPP_INFO(get_logger(), "Setting magnetometer calibration radius");
 
-      port_->write_command_.address = BNO055Register::MAG_RADIUS_LSB;
-      port_->write_command_.length = 2;
-      port_->write_command_.data[0] = magnetometer_radius & 0xFF;
-      port_->write_command_.data[1] = (magnetometer_radius >> 8) & 0xFF;
-      port_->write();
+    port_->write_command_.address = BNO055Register::MAG_RADIUS_LSB;
+    port_->write_command_.length = 2;
+    port_->write_command_.data[0] = magnetometer_radius & 0xFF;
+    port_->write_command_.data[1] = (magnetometer_radius >> 8) & 0xFF;
+    port_->write();
   }
 
   imu_msg_ = std::make_shared<sensor_msgs::msg::Imu>();
   imu_pub_ = this->create_publisher<sensor_msgs::msg::Imu>("imu/data", rclcpp::SensorDataQoS());
   mag_msg_ = std::make_shared<sensor_msgs::msg::MagneticField>();
-  mag_pub_ = this->create_publisher<sensor_msgs::msg::MagneticField>("imu/mag", rclcpp::SensorDataQoS());
+  mag_pub_ = this->create_publisher<sensor_msgs::msg::MagneticField>("imu/mag",
```

Comment:
This violates the [ROS2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#open-versus-cuddled-braces), but it seems to be a defect in `ament_uncrustify`, so I'm okay leaving it as-is for now.

Submitted a PR: https://github.com/ament/ament_lint/pull/148

Reply:
I'll update the package to be compliant regardless of the `nl_func_call_start_multi_line` value.